pkg/go/Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	go clean
 
 test:
-	go test ./...
+	go test ./... -count=1
```

Comment:
why do you need count=1?

Reply:
To disable the cache, otherwise rerunning tests after modifying the test data would only run the previous cached tests